Library/Homebrew/tap.rb

```diff
@@ -31,7 +31,7 @@ def self.fetch(*args)
     repo = repo.strip_prefix "homebrew-"
 
     if user == "Homebrew" && repo == "homebrew" ||
-        user == "Linuxbrew" && repo == "linuxbrew"
+        repo == "linuxbrew"
```

Comment:
This is incorrect. The core tap should always call `Homebrew/homebrew` even for Linuxbrew. Noted this is not the name of Linuxbrew project but  an internal name for a specialize tap.